commands/local_proxy_start.go

```diff
@@ -68,7 +68,7 @@ var localProxyStartCmd = &console.Command{
 			}
 			if err := reexec.Background(varDir); err != nil {
 				if _, isExitCoder := err.(console.ExitCoder); isExitCoder {
-					return err
+					return errors.WithStack(err)
```

Comment:
IIRC `console.ExitCoder` errors must not be wrapped as we have special handling for them in `symfony-cli/console`